src/app/src/pages/eval/components/Eval.tsx

```diff
@@ -142,6 +142,12 @@ export default function Eval({ fetchId }: EvalOptions) {
     // Check for >=1 metric params in the URL.
     const metricParams = searchParams.getAll('metric');
 
+    // Check for >=1 policyId params in the URL.
+    const policyIdParams = searchParams.getAll('policy');
```

Comment:
Reading the policy id from the url when navigating from vuln. data grid on "view logs" btn click.